app/javascript/mastodon/features/status/components/detailed_status.js

```diff
@@ -258,6 +259,15 @@ class DetailedStatus extends ImmutablePureComponent {
       );
     }
 
+    if (status.getIn(['account', 'username']) !== status.getIn(['account', 'acct'])) {
+      externalLink = (
+        <React.Fragment>
+          <React.Fragment> </React.Fragment>
+          <Icon id='external-link' />
```

Comment:
Maybe a `title` and/or `aria-label`?

Reply:
What would they contain?

Reply:
Something like “External link”, “External source” or “Original source”?

Reply:
Added, in what I assume is the correct way but idk really